infra/modules/database/role_manager/role_manager.py

```diff
@@ -3,7 +3,10 @@
 
 
 def lambda_handler(event, context):
-    if event == "check":
-        return check()
-    else:
-        return manage()
+    if event.__class__ is dict and "action" in event and "config" in event:
+        if event["action"] == "check":
+            return check(event["config"])
+        elif event["action"] == "manage":
+            return manage(event["config"])
+    
+    raise Exception("Invalid payload, could not find an action to perform in the lambda handler")
```

Comment:
nit: we don't need to raise an exception here if it's gonna crash anyways. only time we need to raise is if the exception wouldn't make it obvious what was wrong (not the case here) or it wouldn't crash otherwise, which might be the case if `action` exists but isn't "check" or "manage". so we could do something like

```suggestion
    action = event["action"]
    assert action in ("check", "manage")
    if action == "check":
        return check(event["config"])
    elif action == "manage":
        return manage(event["config"])
```

the assert forces a failure if action isn't "check" or "manage" but otherwise if event.__class__ isn't a dict or "action" isn't in event then it would crash anyways so we don't need to explicitly raise